source/includes/_resource.collections.md

```diff
@@ -547,7 +557,7 @@ GET /collections/:id/items
 ```
 
 ```ruby
-Vhx::CollectionItem.list()
+Vhx::Item.all()
```

Comment:
are other client libraries referencing item as a root level resource? they are only represented as a sub resource to collections. thoughts?